stl/inc/array

```diff
@@ -802,6 +807,15 @@ template <class _Ty, size_t _Size>
 _NODISCARD _CONSTEXPR20 bool operator>=(const array<_Ty, _Size>& _Left, const array<_Ty, _Size>& _Right) {
     return !(_Left < _Right);
 }
+#endif // !_HAS_CXX20 || __cpp_impl_three_way_comparison < 201902L
+
+#if defined(__cpp_impl_three_way_comparison) && defined(__cpp_lib_concepts)

```

Comment:
This is probably at least partially my fault for not noticing that EDG had updated `__cpp_impl_three_way_comparison` to `201907`. I've submitted #1049 to cleanup the similarly offending conditionals in `<compare>` and `tests/std/tests/P0768R1_spaceship_operator`.

Reply:
Thanks. While reviewing #1049 I finally understood what Casey was doing, so there's:

## Option 3
* Guard `operator!=` (specifically) with `#if !_HAS_CXX20` (because EDG can synthesize it from `operator==` even without concepts).
* Then guard the other classic `operator<` `operator>` `operator<=` `operator>=` and the new spaceship `operator<=>` with `#ifdef __cpp_lib_concepts`.